cloudinit/util.py

```diff
@@ -1245,19 +1245,44 @@ def close_stdin():
         os.dup2(fp.fileno(), sys.stdin.fileno())
 
 
+def find_devs_with_freebsd(criteria=None, oformat='device',
+                           tag=None, no_cache=False, path=None):
+    if not criteria:
+        return glob.glob("/dev/msdosfs/*") + glob.glob("/dev/iso9660/*")
+    if criteria.startswith("LABEL="):
+        label = criteria.split("LABEL=")[1]
+        devlist = [
+            p for p in ['/dev/msdosfs/' + label, '/dev/iso9660/' + label]
+            if os.path.exists(p)]
+    elif criteria == "TYPE=vfat":
+        devlist = glob.glob("/dev/msdosfs/*")
+    elif criteria == "TYPE=iso9660":
+        devlist = glob.glob("/dev/iso9660/*")
+    return devlist
+
+
 def find_devs_with_netbsd(criteria=None, oformat='device',
                           tag=None, no_cache=False, path=None):
-    if not path:
-        path = "/dev/cd0"
-    cmd = ["mscdlabel", path]
-    out, _ = subp(cmd, capture=True, decode="replace", rcs=[0, 1])
-    result = out.split()
-    if result and len(result) > 2:
-        if criteria == "TYPE=iso9660" and "ISO" in result:
-            return [path]
-        if criteria == "LABEL=CONFIG-2" and '"config-2"' in result:
-            return [path]
-    return []
+    devlist = []
+    label = None
+    _type = None
+    if criteria:
+        if criteria.startswith("LABEL="):
+            label = criteria.split("LABEL=")[1]
```

Comment:
Same, can we use lstript instead of split since we know exactly what we are stripping? Or do you think this is objectionable/inefficient?
```suggestion
            label = criteria.lstrip("LABEL=")
```